firefox-ios/Ecosia/Entitlements/AppExtensions/Ecosia.entitlements

```diff
@@ -10,5 +10,9 @@
 	<array>
 		<string>$(AppIdentifierPrefix)com.ecosia.ecosiaapp</string>
 	</array>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>webcredentials:ecosia.org</string>
+	</array>
```

Comment:
Would this be necessary for the universal links? Like with the forgot password flows?

Reply:
This is going to be necessary by the Auth0 SDK, especially when using https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession